localboot/grub.go

```diff
@@ -78,21 +79,11 @@ func ParseGrubCfg(grubcfg string, basedir string, grubVersion int) []BootConfig
 					// TODO unquote everything, not just \$
 					cmdline = strings.Replace(cmdline, `\$`, "$", -1)
 				}
-				fullpath := path.Join(basedir, kernel)
-				fd, err := os.Open(fullpath)
-				if err != nil {
-					debug("error opening kernel file %s: %v", fullpath, err)
-				}
-				cfg.Kernel = fd
-				cfg.Cmdline = cmdline
+                cfg.Kernel = path.Join(basedir, kernel)
```

Comment:
indentation

Reply:
darn, I committed from a machine without the goimports plugin -.-

Reply:
also.. why doesn't Stickler complain?